terraform/shared/opensearch/data.tf

```diff
@@ -0,0 +1,33 @@
+data "aws_vpc" "folio_rancher" {
```

Comment:
It is better to use the name "this" for unique resources

Reply:
done